packages/shared/src/hooks/useCopyLink.ts

```diff
@@ -39,3 +40,20 @@ export function useCopyLink(
 
   return [copying, copy];
 }
+
+export function useCopyText(text?: string): [boolean, CopyNotifyFunction] {
+  const [copying, setCopying] = useState(false);
+  const { displayToast } = useToastNotification();
+
+  const copy: CopyNotifyFunction = async (props = {}) => {
+    await navigator.clipboard.writeText(props.textToCopy || text);
+    displayToast(props.message || defaultMessage, props);
```

Comment:
This seems off, as the toast would say "Copied link"

I think we need to abstract the useCopyLink a bit to handle generic toast rather if that makes sense

Reply:
I updated the default message and kept both hooks for now as it feels cleaner. I can consolidate these if you prefer though